cmd/client.go

```diff
@@ -125,6 +125,36 @@ func (cli *Client) CreateJobSpec(c *clipkg.Context) error {
 	return cli.deserializeResponse(resp, &jobs)
 }
 
+// CreateJobRun creates job run based on SpecID and optional JSON
+func (cli *Client) CreateJobRun(c *clipkg.Context) error {
+	cfg := cli.Config
+	if !c.Args().Present() {
+		return cli.errorOut(errors.New("Must pass in SpecID, optional JSON param"))
+	}
+	specid := c.Args().First()
+	buf := bytes.NewBufferString("")
+	if c.NArg() > 1 {
+		if !gjson.Valid(c.Args().Get(1)) {
```

Comment:
It'd be great if we could pull out the JSON parsing from the `chainlink create` command, so that this also accepts a file or a JSON blob. Maybe pull that out into a function so it can be used in both places?

Reply:
sure thing! ill get that in later today.